### DISCUSSION TITLE
NEW: Get latest service status information with Amazon Ads Status (beta)

### DISCUSSION THREAD

**Discussion by elizafarley** [amzn/ads-advanced-tools-docs#85] [Announcements]
We recently launched a new tool in beta that lets you check the status for Amazon Ads services, including campaign creation, campaign management, ad delivery, and reporting for sponsored ads and Amazon DSP ([Read the full release note](https://advertising.amazon.com/API/docs/en-us/info/release-notes#keeping-users-informed-about-the-latest-service-status-information-with-amazon-ads-status-beta)).

Amazon Ads Status is available through a [dedicated UI](https://status.ads.amazon.com/), as well as programatically ([JSON](https://status.ads.amazon.com/status.json), [Schema](https://status.ads.amazon.com/status.schema.json)).

Questions or comments? We'd love to hear any feedback!

**Comment by Lucinda2** [comment]
Hi! Can you provide a guide on what am I supposed to do with the JSON? It appears to be a static file: `{"timestamp":"2023-05-08T22:11:12.379Z","language":"english","status":[]}`

**Reply by elizafarley** [reply]
Hi @Lucinda2, thanks for your question! When `status` is empty (`[]`) that means there are no active alerts. If there are active alerts, you will see them listed within the status array, with separate items for region and category. You can refer to [this schema](https://status.ads.amazon.com/status.schema.json) to understand the expected structure of each alert type. 

Hopefully that helps you get started. I have also created a ticket in our documentation backlog to create a guide with examples and best practices when integrating with this tool. Let me know if you have any other feedback!

**Reply by Lucinda2** [reply]
Hi @elizafarley, today's site-wide issue has not been updated to this json, it seems the status is manually maintained which I don't think it'll be a reliable source for us unless it can be automated. We had to get the notice from our users instead of from this portal/json.

e.g. json status
<img width="319" alt="image" src="https://github.com/amzn/ads-advanced-tools-docs/assets/133618719/856ce04a-4abf-4e1b-9596-fb848e171c01">

e.g. ad console notice
<img width="1148" alt="image" src="https://github.com/amzn/ads-advanced-tools-docs/assets/133618719/8ebf0c23-ffa2-4e16-b12a-5740c6bccafc">

**Reply by elizafarley** [reply]
Hi @Lucinda2, thanks for pointing this out! I did some research into this and found that for this particular event, the team decided to display a message only on the advertising console as the impact was limited to data in the past and was meant to be solely informational. 

I have passed on your feedback, and the team will work to ensure more consistency between messaging displayed in the console vs. Ads Status tool in the future.
